bench/workgen/workgen.h

```diff
@@ -342,6 +343,7 @@ struct ThreadOptions {
 	os << "throttle " << throttle;
 	os << ", throttle_burst " << throttle_burst;
 	os << ", synchronized " << synchronized;
+    os << ", session_config " << session_config;
```

Comment:
The spacing looks different from the previous line.

Reply:
It was some weird spacing issue. I had to copy the above os statement and edit it.